tests/test_meshtastic_utils_coverage.py

```
@@ -0,0 +1,926 @@
+import time
+from concurrent.futures import Future, ThreadPoolExecutor
+from unittest.mock import Mock, patch
+
+import pytest
+
+import mmrelay.meshtastic_utils as mu
+from mmrelay.constants.network import METADATA_WATCHDOG_SECS
+
+
+@pytest.fixture(autouse=True)
+def reset_meshtastic_state(reset_meshtastic_globals):
```

Ruff (0.15.4)

[warning] Unused function argument: `reset_meshtastic_globals`

(ARG001)